spark/log4jDebug.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I think such file should be placed in the resource folder. Please check this page: https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html

Reply:
put the file to resource folder

Reply:
why we need this file?